package-lock.json

```diff
@@ -1,10 +1,9 @@
 {
-  "name": "starteki",
```

Comment:
Do we want to change this? I'll leave this as a question for @AMMayberry1

Reply:
I actually changed it to forceteki a while ago but then I did an npm update as part of my last PR and it must've reset it. Good catch